packages/browser-repl/src/components/types/help-output.tsx

```diff
@@ -51,8 +57,9 @@ export class HelpOutput extends Component<HelpOutputProps> {
   render(): JSX.Element {
     const help = this.props.value;
 
+    const className = classnames(styles['help-output']);
```

Comment:
i can't figure out what exactly in your css adds the extra line break between various help items when you do `db.coll1.help()`, but imo, it be nicer if they are a bit more squished together without that extra `\n`

Reply:
I think you'll have to show me what you mean :D